drivers/scheduler/k8s/specs/elasticsearch/aws-storage-class.yaml

```diff
@@ -0,0 +1,8 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: aws-sc-elasticsearch
+provisioner: kubernetes.io/aws-ebs
```

Comment:
Can we have same naming convention for all storage classes?

```
00:31:18.970 INFO[2020-05-15 01:11:09] [elasticsearch] Created storage class: aws-sc-elasticsearch 

00:31:18.974 INFO[2020-05-15 01:11:09] [elasticsearch] Created storage class: azure-es-sc 

00:31:18.984 INFO[2020-05-15 01:11:09] [elasticsearch] Created storage class: px-elasticdata-sc 
```

Reply:
yeah,this is  good point

Reply:
Done